py3status/modules/yubikey.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    format: Display format for the module.
+        (default '\?if=is_waiting YubiKey')
+    gpg_check_timeout: Time to wait for gpg check response.
+        Use value as small as possible that doesn't introduce false positives.
+        (default 0.1)
+    gpg_check_watch_paths: A list of files to watch, start gpg check if any one was opened.
+        (default ['~/.gnupg/pubring.kbx', '~/.ssh/known_hosts'])
+    u2f_check_watch_paths: A list of files to watch, toggle u2f check if any one was opened.
+        (default ['~/.config/Yubico/u2f_keys'])
+
+Control placeholders:
+    {is_waiting} a boolean indicating whether YubiKey is waiting for a touch.
+
+SAMPLE OUTPUT
+{'color': '#00FF00', 'full_text': 'YubiKey'}
+
+Dependencies:
+    gpg: to check for pending gpg access request
+    inotify-simple: to check for pending gpg and u2f requests
+    github.com/maximbaz/pam-u2f: a fork that adds watch capability to pam-u2f module
+    subprocess32: if after all these years you are still using python2
```

Comment:
Modules uses `Requires:` and this `gpg` line might not be needed (installed everywhere).

Not sure if we can use (optional) thing like this.
```
python-inotify-simple: a simple wrapper around inotify
github.com/maximbaz/pam-u2f: (is this optional?) a fork that adds... 
python2-subprocess32: (optional) backported subprocess module for python2
```

Reply:
good ideas:
- Yubikey is bought primarily for gpg operations, so gpg is definitely installed on the system 🙂
- only patched pam-u2f is optional, because that's the only dependency for u2f, will mark it so.